modules/cco-ccoctl-upgrading.adoc

```diff
@@ -22,18 +22,27 @@ On AWS clusters, some `ccoctl` commands make AWS API calls to create or modify A
 
 .Procedure
 
+. From the directory that contains the installation program, set a `$RELEASE_IMAGE` variable with the release image from your installation file by running the following command:
++
+[source,terminal]
+----
+$ RELEASE_IMAGE=$(./openshift-install version | awk '/release image/ {print $3}')
+----
+
 . Extract the list of `CredentialsRequest` custom resources (CRs) from the {product-title} release image by running the following command:
 +
 [source,terminal]
 ----
 $ oc adm release extract \
   --credentials-requests \
-  --cloud=<provider_type> \// <1>
-  --to=<path_to_directory_with_list_of_credentials_requests>/credrequests \// <2>
-  quay.io/<path_to>/ocp-release:<version>
+  --to=<path_to_directory_for_credentials_requests> \// <1>
+  --from=$RELEASE_IMAGE \
+  --included \// <2>
```

Comment:
niy: I think we can drop the `\` here, because this is the final line ine the command:

![image](https://github.com/openshift/openshift-docs/assets/209920/73e48286-8568-42e7-86a1-b115c04a9fcb)

Reply:
oops yep - copy-paste error